tasks/client_config.yaml

```
@@ -1,12 +1,24 @@
 ---
+- name: Check for root password
+  stat:
```

This is not idempotent. Since it's a read-only task we can use `changed_when: false`.
```suggestion
  changed_when: false
  stat:
```